style.css

```diff
@@ -0,0 +1,317 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+#body {
+  background-color: #eee;
+  font-family: 'Lato', sans-serif;
+}
+
+#page-wrapper {
+  position: relative;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.nav-link {}
+
+i1 {
+  font-size: 40px;
+  font-weight: 900;
+  font-family: Bakery;
+}
```

Comment:
Please correct this syntax error on il instead of li, and put Bakery inside a quote.